src/node_getData.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+   class NodeStack {
+     constructor (foo) {
+       this.content = foo;
+     }
+}
+
+
+//
+// nodeA.getData()       // returns the data ("apple") of the node
+// nodeA.setNext(nodeB)  // changes the reference to the next node and returns the original node
+// nodeA.getNext()       // returns the next node, or null if no next node
+
+export default class Stack {
+  const nodeA = new Node({data: "apple"})
```

Comment:
This might be too literal -- it's hard coding in "apple" when it should be able to take anything